components/eam/bld/config_files/definition.xml

```diff
@@ -59,7 +59,7 @@ Switch to turn on/off CLUBB_SGS using clubb to calculate deep: 0 => no, 1 => yes
 <entry id="clubb_do_adv" valid_values="0,1" value="0">
 Switch to turn on/off advecting CLUBB moments: 0 => no, 1 => yes
 </entry>
-<entry id="pbl" valid_values="uw,hb,hbr,clubb_sgs" value="">
+<entry id="pbl" valid_values="uw,hb,hbr,clubb_sgs,none" value="">
```

Comment:
Likewise, does "none" mean disable, or select a default for me?

Reply:
added a comment for these in upcoming commits.